src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1318,8 +1321,7 @@ func (k *kataAgent) createContainer(ctx context.Context, sandbox *Sandbox, c *Co
 		return nil, err
 	}
 
-	ctrStorages = append(ctrStorages, volumeStorages...)
-
+	ctrStorages = append(volumeStorages, ctrStorages...)
```

Comment:
As we have discussed offline, this change will break support of data volume.
We need to refine the overall design of patch 2 by:
1) hook createDevices() to parse and create devices for `KataVirtualVolume`
2) hook ShareRootFilesystem() to parse and create `Storage` devices for rootfs with `KataVirtualVolume`.
3) later we can also hook handleBlkOCIMounts() to support `KataVirtualVolume` for verity/encrypt data volumes

Reply:
Done.